dsfs/package.go

```diff
@@ -50,6 +50,8 @@ const (
 	PackageFileMeta
 	// PackageFileViz isolates the data related to representing a dataset as a visualization
 	PackageFileViz
+	// PackageFileRendered is the rendered visualization of the dataset
+	PackageFileRendered
```

Comment:
how about `PackageFileRenderedViz`? Sounds pedantic, but it at least confirms _what_ is being rendered.